automated/utils/test-runner.py

```diff
@@ -17,6 +18,17 @@
 from distutils.spawn import find_executable
 
 
+try:
+    import requests
```

Comment:
I think requests is not needed. squad-client has all dependencies.

Reply:
Removed